raiden/raiden_service.py

```diff
@@ -635,6 +651,214 @@ def settle(self, token_address, partner_address):
         return channel
 
 
+class ConnectionManager(object):
+    """
+    The ConnectionManager provides a high level abstraction for connecting to a
+    Token network.
+    Note:
+        It is initialized with 0 funds; a connection to the token network
+        will be only established _after_ calling `connect(funds)`
+    """
+    # XXX Hack: for bootstrapping the first node on a network opens a channel
+    # with this address to become visible.
+    BOOTSTRAP_ADDR = '2' * 40
+    BOOTSTRAP_ADDR_BIN = BOOTSTRAP_ADDR.decode('hex')
```

Comment:
shouldn't we remove the `_BIN` suffix here and add `_HEX` to the variable above instead as per our new naming policy?